scenes/game/game.gd

```diff
@@ -1741,7 +1743,13 @@ func begin_strike_choosing(strike_response : bool, cancel_allowed : bool,
 	select_card_require_min = 1
 	select_card_require_max = 1
 	var can_cancel = cancel_allowed and not strike_response
-	enable_instructions_ui("Select a card to strike with.", true, can_cancel, not disable_wild_swing, not disable_ex)
+	var dialogue = "Select a card to strike with."
+	var player = game_wrapper._get_player(Enums.PlayerId.PlayerId_Player)
```

Comment:
so, I pretty explicitly tried to not directly call things internal to LocalGame/Player from Game as an effort to keep the UI and game logic separate.

Can you add a function to game_wrapper for what you're trying to check?
Something like game_wrapper.get_sidestep_card_name() or something? and then add it to dialogue if it is not empty?